Makefile

```diff
@@ -3,7 +3,7 @@ all:
 	@echo "Generate all the things"
 
 #	general code generation for all the supported languages
-	buf generate proto --template buf.gen.all.yaml --verbose
+	buf generate proto --template buf.gen.all.yaml --verbose --include-imports
```

Comment:
I see that this generated some extra stuff for Go and Java... and I have a bad feeling that this might result in some confused dependencies - but I'll let you first investigate the issue with Java. However, in case it is this, and also in the spirit of keeping the changes to a minimum, I'd suggest:
- removing this `--include-imports` flag from here
- further, on line 35 where you're doing the `buf generate proto --template buf.gen.pyi.yaml --include-imports`, you could rename that file to `buf.gen.python.yaml`.
- then, take the python plugin out of the `buf.gen.all.yaml` and move it to this new file, together with the `pyi` plugin and keep the `--include-imports` flag only with this python generation.

If I'm not mistaken, like this we could generate the files for python and adding the imports because it needs them, but avoid polluting java and go with them.